apps/desktop/src/lib/components/BranchPreview.svelte

```diff
@@ -225,4 +225,18 @@
 	.card__content {
 		color: var(--clr-scale-ntrl-30);
 	}
+
+	.branch-group {
+		border: 1px solid var(--clr-border-2);
+		border-radius: var(--radius-m);
+		background: var(--clr-bg-1);
+
+		&:last-child {
+			margin-bottom: 12px;
+		}
+
+		& :global(.commit-row):first-child {
+			border-radius: var(--radius-m) var(--radius-m) 0 0;
+		}
+	}
```

Comment:
Yeah so this just adds the border and bg color and a few things that are missing in this context because in this branchpreview view we don't the rest of the lane wrapping it, or the lane header (therefore the additional radius).